test/SymbolGraph/Symbols/Mixins/SPI.swift

```diff
@@ -1,19 +1,22 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-build-swift %s -module-name SPI -emit-module -emit-module-path %t/
+// RUN: %target-build-swift %s -module-name SPI -emit-module -emit-module-path %t/SPI.swiftmodule -include-spi-symbols
 // RUN: %target-swift-symbolgraph-extract -module-name SPI -I %t -pretty-print -output-dir %t -include-spi-symbols
 // RUN: %FileCheck %s --input-file %t/SPI.symbols.json --check-prefix SPI
+// RUN: %FileCheck %s --input-file %t/SPI.symbols.json --check-prefix SPIDOC
```

Comment:
Is there a specific reason for preferring a separate `RUN:` line here (and at line 13)? Multiple prefixes can be combined in the same `FileCheck` command: `--check-prefix PRE1 --check-prefix PRE2`.

Reply:
Will specifying multiple check prefixes create ordering constraints? I've run into trouble in the past trying to squeeze all my checks into a single prefix because of how the symbol graph doesn't have a consistent ordering.

Reply:
Yes it will create ordering constraints. You can use `CHECK-DAG` if you want to have mixed order, although, sometimes it's not quite as flexible as one would like for JSON, where you'd like to do nested checks.

I think it's fine to leave this in if `CHECK-DAG` is not good enough.